### ISSUE TITLE
Uncaught exception when opening "About Rocket.Chat+"

### ISSUE THREAD

**Issue by msbentley** [RocketChat/Rocket.Chat.Electron#467]
## My Setup
- Operating System: Ubuntu 17.04
- App Version: 2.7.0
- [x] I have tested with the latest version
- [x] I can simulate the issue easily

## Description
When opening the About Rocket.Chat+ menu option the "About" window opens, empty, and I also get an error window "A JavaScript error occurred in the main process" with the following content:

```Uncaught Exception:
TypeError: Error processing argument at index 0, conversion failure from undefined
    at Event.set (/opt/Rocket.Chat+/resources/electron.asar/browser/api/web-contents.js:252:22)
    at EventEmitter.electron.ipcMain.on (/opt/Rocket.Chat+/resources/app.asar/app/background.js:496:27)
    at emitTwo (events.js:106:13)
    at EventEmitter.emit (events.js:191:7)
    at WebContents.<anonymous> (/opt/Rocket.Chat+/resources/electron.asar/browser/api/web-contents.js:256:37)
    at emitTwo (events.js:106:13)
    at WebContents.emit (events.js:191:7)
```
I believe this did not happen with previous versions, but started with 2.7.0.

**Comment by lindyhopchris** [comment]
For info, I'm getting exactly the same on Ubuntu 16.04

**Comment by TriPhoenix** [comment]
For our deployment this issue disappeared when updating to version 2.8.0, maybe the fix is already there?

**Comment by msbentley** [comment]
Hmm, I've just installed the 2.8.0 64-bit .deb and I still get the same issue here...

**Comment by ml48603** [comment]
I went to the website and looked for 2.8.0, its not there.  Where did you get your x64 2.8.0 deb ?
https://rocket.chat/download
Tried: https://github.com/RocketChat/Rocket.Chat.Electron/releases/download/2.8.0/rocketchat-desktop_2.8.0_amd64.deb

**Comment by gdelavald** [comment]
Website is going to be updated soon with the links, you can find in the releases on this repo.
Here you go:
https://github.com/RocketChat/Rocket.Chat.Electron/releases/download/2.8.0/rocketchat_2.8.0_amd64.deb

**Comment by ml48603** [comment]
Thanks gdelavald , installing now !  Will see if they fixed that About Issue on my install.  Are their release notes for the linux client?  Where are the release notes posted ?

**Comment by ml48603** [comment]
The about works now in 2.8.0 x64 deb.  Even got a nice update checker.  That's what the doctor ordered.  Nice job RC Developers ! Thank you.
![image](https://user-images.githubusercontent.com/17936105/28172646-1d8e7c04-67ba-11e7-9bd7-a6142054aa03.png)

**Comment by ml48603** [comment]
msbentley
I have two identical computers with two nearly identical installs of Ubuntu 16.04.2 Desktop AMD64

RC Desktop 2.8.0 About Dialogue worked perfect on the first.  On the other PC it did not work.  I then started again and followed the same procedure on the 2nd PC I followed on the first.

1. Logout of each team.
2. Use xkill to kill the 2.7.0 process if upgrading from 2.7.0. to 2.8.0. Can do same for reinstallation of 2.8.0.
3. Run 2.7.0 deb and use GDebi Package Manager to uninstall 2.7.0
4. Install 2.8.0
5. Try About dialogue (Should work now)
6. Log in to teams again.

When I quit without logging into the teams the new installation was crippled. (About didnt work)  Wonder what else doesn't install correctly when the user does not logout ?
